bucket/inbucket.json

```diff
@@ -0,0 +1,38 @@
+{
+    "homepage": "https://www.inbucket.org/",
+    "description": "Inbucket is an email testing application; it will accept messages for any email address and make them available to view via a web interface.",
+    "license": "MIT",
+    "version": "2.1.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/inbucket/inbucket/releases/download/v2.1.0/inbucket_2.1.0_windows_amd64.zip",
+            "hash": "627838af6d1ecc50a19960f2a92fdebfd7d7accb0daf85fd7bc3cb9d011dc109",
+            "extract_dir": "inbucket_2.1.0_windows_amd64",
+            "bin": [
+                "inbucket.exe",
+                "inbucket-client.exe"
+            ]
+        }
+    },
+    "env_set": {
+        "INBUCKET_SMTP_TLSPRIVKEY": "$dir\\cert\\cert.key",
+        "INBUCKET_SMTP_TLSCERT": "$dir\\cert\\cert.crt",
+        "INBUCKET_WEB_UIDIR": "$dir\\ui",
+        "INBUCKET_WEB_GREETINGFILE": "$dir\\ui\\greeting.html"
+    },
+    "persist": "cert",
+    "checkver": {
+        "github": "https://github.com/inbucket/inbucket"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/inbucket/inbucket/releases/download/v$version/inbucket_$version_windows_amd64.zip",
+                "hash": {
+                    "url": "https://github.com/inbucket/inbucket/releases/download/v$version/inbucket_$version_checksums.txt"
```

Comment:
```suggestion
                    "url": "$baseurl/inbucket_$version_checksums.txt"
```

Reply:
Done

Reply:
Do not write "done". Mark conversations as done, for automatic collapsing of discussions. No need to scroll through outdated discussions.